### DISCUSSION TITLE
Spanish audio - no transcription in the response

### DISCUSSION THREAD

**Discussion by damianherrera** [deepgram/community#158] [General help] [answered]
Updated question: I am providing 58 second audio is in Spanish and I am not receiving any text.

```json
{
    "metadata": {
        "transaction_key": "deprecated",
        "request_id": "96057111-d82f-4e94-ae74-8617851ec64d",
        "sha256": "e9f1b2c2c86413bbad555a8a502e5a611be699190f84d1ea4f35758262ea5ad4",
        "created": "2023-05-18T13:22:56.576Z",
        "duration": 58.440002,
        "channels": 1,
        "models": [
            "96a295ec-6336-43d5-b1cb-1e48b5e6d9a4"
        ],
        "model_info": {
            "96a295ec-6336-43d5-b1cb-1e48b5e6d9a4": {
                "name": "general",
                "version": "2023-02-22.3",
                "arch": "base"
            }
        }
    },
    "results": {
        "channels": [
            {
                "alternatives": [
                    {
                        "transcript": "",
                        "confidence": 0.0,
                        "words": []
                    }
                ]
            }
        ]
    }
}
```

**Comment by ajsyp** [comment]
Nope, you're doing everything just fine. The `transaction_key` has nothing to do with pre-recorded audio. It was an old debug field to help us trace issues through our system and provide faster support. We've replaced that with other automated tools these days. We'll remove it entirely in a future API version.

**Comment by damianherrera** [comment]
Thank you very much for your prompt response. Can I call the API in some way to transcribe the pre-recorded audio?

**Comment by ajsyp** [comment]
You're using the right API call for pre-recorded. It isn't included in your screen shot, but that `results` section at the bottom of your screenshot should contain the transcript, words, confidences, times, etc.

In other words, the `transaction_key` **field** is deprecated, but not the entire API call. You're still using the correct, latest-and-greatest endpoint for pre-recorded audio.

**Comment by damianherrera** [comment]
Thank you very much for your quick response. I give you the result. The 58 second audio is in Spanish and I am not receiving any text.

```
{
    "metadata": {
        "transaction_key": "deprecated",
        "request_id": "96057111-d82f-4e94-ae74-8617851ec64d",
        "sha256": "e9f1b2c2c86413bbad555a8a502e5a611be699190f84d1ea4f35758262ea5ad4",
        "created": "2023-05-18T13:22:56.576Z",
        "duration": 58.440002,
        "channels": 1,
        "models": [
            "96a295ec-6336-43d5-b1cb-1e48b5e6d9a4"
        ],
        "model_info": {
            "96a295ec-6336-43d5-b1cb-1e48b5e6d9a4": {
                "name": "general",
                "version": "2023-02-22.3",
                "arch": "base"
            }
        }
    },
    "results": {
        "channels": [
            {
                "alternatives": [
                    {
                        "transcript": "",
                        "confidence": 0.0,
                        "words": []
                    }
                ]
            }
        ]
    }
}
```

**Comment by ajsyp** [comment, marked as answer]
Ah, that's because the default language is English and it isn't detecting any English words. Try switching to Spanish by using this URL: `https://api.deepgram.com/v1/listen?language=es`. You might want to try the enhanced model, too: `https://api.deepgram.com/v1/listen?language=es&model=enhanced` (support coming for Nova soon).

We also have a language detection feature that you could try: `https://api.deepgram.com/v1/listen?detect_language=true`.

For more information, check out [our documentation](https://developers.deepgram.com/api-reference/transcription/).

**Reply by damianherrera** [reply]
Thank you very much!
